shared/folders/render.desktop.js

```diff
@@ -45,6 +50,10 @@ class Render extends Component<void, Props, State> {
   }
 
   render () {
+    if (this.props.allowComingSoon && !flags.tabFoldersEnabled) {
```

Comment:
Would it make sense to put this feature flagging logic in the smart components rather than plumbing this logic through dumb components? The necessity of `allowComingSoon` to keep this out of the dumb sheet is a smell. It seems to me like the feature flags are an externality to the dumb components, and they should only focus on rendering specific to their purpose.